unskript-ctl/unskript_ctl_notification.py

```diff
@@ -207,24 +207,28 @@ def create_script_summary_message(self, output_metadata_file: str):
             self.logger.error(f'ERROR: Metadata is empty for the script. Please check content of {output_metadata_file}')
             raise ValueError("Metadata is empty")
 
-        message += f'''
-                <br>
-                <h3> Custom Script Run Result </h3>
-                <table border="1">
-                    <tr>
-                        <th> Status </th>
-                        <th> Time (in seconds) </th>
-                        <th> Error </th>
-                    </tr>
-                    <tr>
-                        <td>{metadata.get('status')}</td>
-                        <td>{metadata.get('time_taken')}</td>
-                        <td>{metadata.get('error')}</td>
-                    </tr>
-                </table>
-        '''
-
-        return message
+        # Log the custom script run result instead of including it in the email.
+        self.logger.debug(f"\tStatus: {metadata.get('status')} \n\tTime (in seconds): {metadata.get('time_taken')} \n\tError: {metadata.get('error')} \n")
+
+        # Remove from email 
+        # message += f'''
+        #         <br>
+        #         <h3> Custom Script Run Result </h3>
+        #         <table border="1">
+        #             <tr>
+        #                 <th> Status </th>
+        #                 <th> Time (in seconds) </th>
+        #                 <th> Error </th>
+        #             </tr>
+        #             <tr>
+        #                 <td>{metadata.get('status')}</td>
+        #                 <td>{metadata.get('time_taken')}</td>
+        #                 <td>{metadata.get('error')}</td>
+        #             </tr>
+        #         </table>
+        # '''
+
+        return ''
```

Comment:
This also feels like something to be included in the `email_fmt` settings. Lets discuss in our meeting later today.